en/getting_started/frame_selection.md

```diff
@@ -24,6 +24,20 @@ process](../config/README.md).
 
 ![Frame Selection](../../images/frame_selection.png)
 
+<!-- 

```

Comment:
@tubeme This is the content on types of VTOL you had in the modes docs. IMO we do need to keep it, but it belongs in the Frames docs. Unfortunately it doesn't "Fit" in the current structure of that doc, so for now I am just putting it into that doc as a comment so we can come back to it. OK?

Reply:
I think that the transition is part of the flying and modes because from the pilots perspective it is like a mode switch. Also it is called FW mode and MC mode. But you cannot explain about transition except if you explain what make the difference in the transition. So I think that it does not hurt if the modes and the transition are explained in this doc. Probably the frame doc will be ok... (BTW our frame that we are developing is VTOL and the transition is one of the most important concept that the pilot should grasp.)

Reply:
> I think that the transition is part of the flying and modes because from the pilots perspective it is like a mode switch. 

Agreed. The actual selection item in QGC needs to be mentioned here.

> Also it is called FW mode and MC mode. 

True, but that is horrible to parse when we already refer to mode for "flight modes". I have tried to avoid repeating that terminology.

> But you cannot explain about transition except if you explain what make the difference in the transition. So I think that it does not hurt if the modes and the transition are explained in this doc. 

This is where I disagree. Somewhere we need to explain the differences, but not in what is effectively a high level intro for beginners. I think we need to talk about the different types in this (airframe intro) doc.

I'll think about how the section can be extended to address your points.